doc/src/stdlib/test.md

```diff
@@ -169,46 +169,7 @@ As calculations on floating-point values can be imprecise, you can perform appro
 checks using either `@test a ≈ b` (where `≈`, typed via tab completion of `\approx`, is the
 [`isapprox()`](@ref) function) or use [`isapprox()`](@ref) directly.
 
-An alternative is the `@test_approx_eq` macro (which differs from [`isapprox`](@ref) in that it treats
-`NaN` values as equal and has a smaller default tolerance) or `@test_approx_eq_eps` (which takes
-an extra argument indicating the relative tolerance):
-
-```julia
-julia> @test 1 ≈ 0.999999999
-
-julia> @test 1 ≈ 0.999999
-ERROR: test failed: 1 isapprox 0.999999
- in expression: 1 ≈ 0.999999
- in error at error.jl:21
- in default_handler at test.jl:30
- in do_test at test.jl:53
```

Comment:
about to get on a plane (aren't you, too?), I have a commit on a misc branch locally to put these back, can go into a PR next time I rearrange branches